### PR TITLE
Feature/top menu

### PR DESCRIPTION
**Pull request by vojislavgvozdic**
Added Demos in the top menu after Technical Reference, along with all the Demo files Sladja created.

### CODE REVIEW

File: docs/demos/zdm/Connect_Zerynth_Device_Manager_with_Grafana_for_IoT_data_Visualization.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+![](img/add-querry.png)
+
+Now, to open the Select Metric Menu and choose the ZDM Tag that is going to be used as an IOT data source for the visualization the user is building.
+
+If any tag can’t be seen, it means that the user doesn't have any data in the ZDM storage and no devices streaming. To launch a Python client device this [guide](https://docs.zerynth.com/latest/deploy/getting_started_with_rpi/) can be used, to connect a Zerynth IOT device this [guide](https://www.zerynth.com/blog/docs/zdm/getting-started/the-zdm-web-user-interface/) can be followed.
+
+Here it is,a ZDM data plotted in Grafana!
+
+![](img/ZDMdemo.png)
```
Comment (ghost) [inline docs/demos/zdm/Connect_Zerynth_Device_Manager_with_Grafana_for_IoT_data_Visualization.md]: This image has to be moved into the powerbi demo with the ZDM. (Connect Power Bi to the Zerynth Device Manager, docs/demos/zdm/Connect_Power_Bi_to_the_Zerynth_Device_Manager.md )

File: docs/demos/zdm/Connect_Power_Bi_to_the_Zerynth_Device_Manager.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    Table.ExpandRecordColumn(myTableRec, "Value", { "tag", "timestamp_device", "device_id", "payload" })
+```
+​
+The wizard can be opened by clicking on “split” in the payload column of the new table you will get.
+
+![](img/wizard.png)
+
+Now the user can publish their Dashboard online if they want!
+
+![](img/dashboard.png)
```
Comment (ghost) [inline docs/demos/zdm/Connect_Power_Bi_to_the_Zerynth_Device_Manager.md]: This image has to be moved into the Grafana Demo (Connect Zerynth Device Manager with Grafana for IoT data Visualization- docs/demos/zdm/Connect_Zerynth_Device_Manager_with_Grafana_for_IoT_data_Visualization.md ).

**Review by ghost — CHANGES_REQUESTED** [review]
Could you please fix the two screenshots of the ZDM demos ?

File: docs/demos/zdm/Connect_Zerynth_Device_Manager_with_Grafana_for_IoT_data_Visualization.md (hunk 1 above)
Comment (vojislavgvozdic) [reply docs/demos/zdm/Connect_Zerynth_Device_Manager_with_Grafana_for_IoT_data_Visualization.md]: Image moved

File: docs/demos/zdm/Connect_Power_Bi_to_the_Zerynth_Device_Manager.md (hunk 2 above)
Comment (vojislavgvozdic) [reply docs/demos/zdm/Connect_Power_Bi_to_the_Zerynth_Device_Manager.md]: image moved

**Review by ghost — APPROVED** [review]

**Pull request merged**